Samples/Client/WebClient/README.md

```diff
@@ -0,0 +1,45 @@
+The WebClient sample is a vanilla JavaScript sample that supports mono rendering with input handlers for moving/rotating a 3D camera.
+
+## Installing dependencies
+
+The WebClient sample consumes a universal JavaScript plugin that is downloaded that is downloaded using npm. The source code can be found [here](https://github.com/CatalystCode/js-3dtoolkit). Installation is done using the `npm install` command.
+
+```bash
+$ npm install
```

Comment:
nit: i think this would be better if it was `npm install js-3dtoolkit`

Reply:
we already have a package.json file in the folder, so, npm install will install js-3dtoolkit and all other dependencies. I will make that clear in the README